tests/unit_tests/PreprocessorTests.cpp

```diff
@@ -194,7 +194,7 @@ TEST_CASE("Function macro (keyword as formal argument)", "[preprocessor]") {
 }
 
 TEST_CASE("Macro pasting (identifiers)", "[preprocessor]") {
-    auto& text = "`define FOO(x,y) x   ``   _blah``y\n`FOO(   bar,    _BAZ)";
+    auto& text = "`define FOO(x,y) x``_blah``y\n`FOO(   bar,    _BAZ)";
```

Comment:
My original reason was I realized that it the compilers I tried on EDA playground did not have this feature.  HRT code does rely on whitespace existing next to a macro paste operator (e.g. hrt_prologue.svh:182).  The main reason I prefer this way is that it means you don't have to think of the operator as a paste between two neighboring tokens but instead you can just surround your argument with this operator regardless of its context to treat it more like an escaping mechanism.  This idea of putting the operator on both sides of the argument even when it isn't necessary is heavily used in our codebase.